modules/standard/Reflection.chpl

```diff
@@ -220,46 +331,96 @@ proc getFieldRef(ref x:?t, param s:string) ref {
   return __primitive("field by num", x, i);
 }
 
+/* Get a field index in a class or record, or -1 if
```

Comment:
I think `-1` should be surrounded by two backtics on either side so that it pops nicely in the generated documentation